### PR TITLE
Remove cloud-connector dependency + deploy key when deploying Pixie from OpApps operator

### PR DESCRIPTION
**Pull request by ginwoopak**

### CODE REVIEW

File: bin/px
Comment (ginwoopak) [inline bin/px:1]: Remove. no longer used.

File: operator/config/manager/kustomization.yaml
Comment (ginwoopak) [inline operator/config/manager/kustomization.yaml:1]: revert back.

File: operator/hack/autoinstrumentation/generate_yamls.sh [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
+echo -e "---\n$(cat vizier_deps/nats_prod.yaml)" > vizier_deps/nats_prod.yaml
+files_to_apply=(vizier/vizier_metadata_persist_prod.yaml vizier/secrets.yaml vizier_deps/nats_prod.yaml)
 cat "${files_to_apply[@]}" | csplit -n 3 -f 'splits/autoinstrumentation-' - '/^---$/' "{$(($(cat "${files_to_apply[@]}" | grep -c '^\-\-\-$') - 2))}"
 
 # Remove duplicate resources
-duplicates=$(fdupes -f splits)
-if [[ $duplicates != "" ]]; then
-  echo "$duplicates" | grep -v '^$' | xargs rm -fv
-fi
+#duplicates=$(fdupes -f splits)
```
Comment (ginwoopak) [inline operator/hack/autoinstrumentation/generate_yamls.sh:24]: Delete commented code.

File: operator/hack/autoinstrumentation/generate_yamls.sh [hunk 2]
```diff
@@ -54,10 +66,15 @@ mkdir -p "${REPO_ROOT}/operator/deploy/internal/autoinstrumentation"
 cp splits/secrets/*.yaml "${REPO_ROOT}/operator/deploy/internal/autoinstrumentation"
 cp splits/*.yaml "${REPO_ROOT}/operator/deploy/internal/autoinstrumentation"
 
-sed -i '' 's/  PL_CLUSTER_NAME: "replace_me"/  PL_CLUSTER_NAME: {{ .ClusterName }}/' "${REPO_ROOT}/operator/deploy/internal/autoinstrumentation/00-configmap-pl-cloud-config.yaml"
-sed -i '' 's/  deploy-key: "replace_me"/  deploy-key: {{ .Experimental.AutoInstrumentation.DeployKey }}/' "${REPO_ROOT}/operator/deploy/internal/autoinstrumentation/01-secret-pl-deploy-secrets.yaml"
+#find "${REPO_ROOT}"/operator/deploy/internal/autoinstrumentation/ -type f -name '*.yaml' -exec sed -i '' 's/          image: gcr.io/          image: projects.registry.vmware.com\/asap/' {}
```
Comment (ginwoopak) [inline operator/hack/autoinstrumentation/generate_yamls.sh:69]: remove

**Review by m25n — APPROVED** [review]

**Pull request merged**